Dockerfile

```diff
@@ -31,12 +31,16 @@ RUN apt-get update -y && \
     apt-get clean && \
 	rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-RUN echo "Downloading Connect IQ SDK: ${VERSION}" && \
+RUN echo "Downloading Connect IQ SDK: ${VERSION} from ${CONNECT_IQ_SDK_URL}/sdks/connectiq-sdk-lin-${VERSION}.zip" && \
     cd /opt && \
     curl -LsS -o ciq.zip ${CONNECT_IQ_SDK_URL}/sdks/connectiq-sdk-lin-${VERSION}.zip && \
     unzip ciq.zip -d ciq && \
     rm -f ciq.zip
 
+# fetched devices for 4.0.1
+RUN mkdir -p /home/developer/.Garmin/ConnectIQ/Devices/
+ADD devices.tar.xz /home/developer/.Garmin/ConnectIQ/Devices/
```

Comment:
I fetched the devices files using `sdkmanager` for Windows. Then compress those into `devices.tar.xz`.